src/main/java/seedu/academydirectory/model/student/Assessment.java

```diff
@@ -1,11 +1,22 @@
 package seedu.academydirectory.model.student;
 
+import java.util.Arrays;
+import java.util.Comparator;
 import java.util.HashMap;
+import java.util.List;
 
 /**
  * Represents a Student's assessment in the academy directory.
  */
-public class Assessment implements Information {
+public class Assessment extends SortableWrapper {
```

Comment:
Err maybe SortableInformation instead? Cuz this seems like Assessment is a SortableWrapper which sounds a bit strange. Alternatively maybe have a Sortable<Assessment> for example used in various places in the code?

Reply:
Wait there seems to be a SortableInformation alr?